pkg/pillar/cmd/zedrouter/zedrouter.go

```diff
@@ -1618,6 +1720,84 @@ func doAppNetworkModifyUnderlayNetwork(
 	publishNetworkInstanceStatus(ctx, netstatus)
 
 	maybeRemoveStaleIpsets(staleIpsets)
+	return nil
+}
+
+// returns a interfaces list, with network change
```

Comment:
Reading this comment it makes we guess it returns a list of interface names, but that clearly isn't the case.
So the comment (and potentially the naming) needs to be improved.
It seems like it is a changedUlIndexList of booleans where the index is the index in the config.UnderlayNetworkList.